nf_core/pipeline-template/.github/workflows/template_version_comment.yml

```diff
@@ -0,0 +1,42 @@
+name: nf-core template version comment
+# This workflow is triggered on PRs to check if the pipeline template version matches the latest nf-core version.
+# It posts a comment to the PR, even if it comes from a fork.
+
+on: pull_request_target
```

Comment:
not sure how strict we should be with this comment (don't want to spam too much). How about only posting it for PRs against `master`?

Reply:
I see it can get too annoying, but I think only on PRs to master is too little, we want template updates as soon as possible. 
Right now, linting tests are failing if the template is not updated, so I think it's already an improvement to have green tests at the cost of a comment on every PR.

Reply:
yes, that's true. My fear is just that warnings that are coming too often have a tendency to be ignored. Maybe something you could discuss with @ewels in the next devcore meeting?

Reply:
Let's be as annoying as possible 🎉

Reply:
We should include a gif with the comment which slowly escalates proportionally according to the time elapsed since the pipeline was updated.

Start of slowly and subtly and end with huge flashing annoying gifs 😆